.github/README.md

```diff
@@ -21,3 +21,25 @@ Dette vil automatisk laste inn nye endringer når du lagrer filer, noe som gjør
 
 ### Kjøre testene:
 *Ikke implementert*
+
+### MariaDb database som Docker konteiner
+
+1. Opprett MariaDb konteiner.
+  
+
+```docker
+docker run --name NordicDoor -e MYSQL_ROOT_PASSWORD=PASSORD -p 3306:3306 -d docker.io/library/mariadb:latest
```

Comment:
Kanskje endre dette til et mer beskrivende navn? Som `db`eller `database`?

```suggestion
docker run --name db -e MYSQL_ROOT_PASSWORD=PASSORD -p 3306:3306 -d docker.io/library/mariadb:latest
```

Reply:
Hva databasen heter spiller ikke så stor rolle for meg, men tenker at det er greit at den har et navn som tilsier at den tilhører NordicDoor applikasjonen.

Hvis den blir endret må vi huske å endre connection string i PDSA_System/Server/appsettings.json

Reply:
Ja er enig med @tbhansen Burde hete NordicDoor eller noe sånt. Fordi da når vi ser gjennom databasene våre i DataGrip, då blir det lettere så vite hva dem databasen tilhører

Reply:
@sdushantha Navnet på containeren vil ikke være synlig for datagrip, og det er det jeg tenker bør beskrive hva det kjører, og ikke hvem det tilhører. Derfor synes jeg at `db` er mye mer passende. Så kan vi heller bruke `NordicDoor` som navn på dotnet-containeren

Reply:
@menymo Du har et godt poeng Magnus! Jeg trodde du mente at vi skulle kalle databasen for `db` når jeg først leste kommentaren din, men skjønner nå at du mener mariadb konteineren.

Reply:
@menymo samme som tobias, leste feil 😅
vi kan jo godt kalle det for `db` eller `mariadb`

Reply:
@sdushantha @menymo 

**Forslag**
En konteiner med navn `dbCon`, som inneholder to databaser med navn `ProdDb` og `DevDb`.

**Forkortelser**
DbCon = Database Container
ProdDb = Production Database
DevDb = Development Database (til testing)

Kanskje vi bør bruke noe annet enn slik camelcasing.
Disse tankene kom etter å ha lest denne siden: https://carsandcode.com/2020/05/02/naming-oracle-container-databases/

Reply:
Hvorfor trenger vi en Prod og Dev database?

> dbCon

Personlig foretrekker jeg `db` , `database` eller `mariadb`

Reply:
Mulig det blir unødvendig komplisert. Dersom noen ønsker å lage en ekstra database for testing så står man forsåvidt fritt til å gjøre det på egenhånd. 
Tror det viktigste er at navnet gir et klart skille på hva som er konteiner og hva som er database.

Reply:
Jeg tror `Con` blir litt redundant for navnet til en container. Om noen ønsker å lage en database til utvikling, så er det helt mulig å endre på første linjen på `.sql`-filen og connection string i `appsettings.json`, det kan være lurt senere i utviklingen før vi er klare til å sende avgårde prosjektet